### DISCUSSION TITLE
What is options.displayFields in the relation schema and how it's being populated?

### DISCUSSION THREAD

**Discussion by mrbu-solutions** [pocketbase/pocketbase#2584] [Q&A] [answered]
What is options.displayFields in the relation schema and how it's being populated?

![image](https://github.com/pocketbase/pocketbase/assets/53817867/f4973b20-89ab-42e8-b464-be19db222b9d)

**Comment by ganigeorgiev** [comment, marked as answer]
It's a strings array of the relation field names to use in the Admin UI when listing the relation (eg. `name`, `title`, etc.). If it is empty, the Admin UI tries to search for some fixed commonly used fields and if none are found fallbacks to the `id`.

In future version this option will be moved in the collection model so that the collection displayable fields are defined only once.

**Reply by mrbu-solutions** [reply]
Thank you very much.
